api/tests/connectors/beneficiaries/ubble_test.py

```diff
@@ -105,6 +105,20 @@ def test_use_real_ubble_if_ff_enabled_and_url_unset(self):
 
         assert url == "https://api.ubble.ai"
 
+    @override_features(WIP_ENABLE_MOCK_UBBLE=True)
+    @override_settings(UBBLE_MOCK_API_URL="http://example.com/partial/path")
+    def test_concat_full_path_if_using_mock(self):
+        url = ubble.build_url("/and/end")
+
+        assert url == "http://example.com/partial/path/and/end"
+
+    @override_features(WIP_ENABLE_MOCK_UBBLE=False)
+    @override_settings(UBBLE_API_URL="https://api.ubble.ai/prefix")
+    def test_use_base_url_if_not_using_mock(self):
+        url = ubble.build_url("/and/end")
+
+        assert url == "https://api.ubble.ai/and/end"
```

Comment:
C'est voulu de ne pas avoir "/prefix/and/end" ? Le comportement est différent de `UBBLE_MOCK_API_URL` c'est plutôt fourbe, non ?

Reply:
Oui c'est voulu, c'est vrai que je peux l'expliciter dans le test en changeant `prefix` par `not-to-be-used-prefix` par exemple.
C'est un peu fourbe aussi j'en conviens, mais ça nous arrange de préfixer les chemins des URLs du mock pour mieux définir les accès par route. [Voir cette PR](https://github.com/pass-culture/mock-ubble/pull/12/files).
Or, `urljoin` ne va pas tenir compte de ce préfixe, c'est pour ça que j'ai changé la méthode de concaténation.
Je suis preneur d'une meilleur idée :)

Reply:
> EDIT: Sinon on peut utiliser une concaténation simple (avec un +) dans les deux cas. Ça conviendrait mieux ?

Oui, je pense que ce serait plus clair. Avec une ceinture de sécurité pour bien gérer les cas où on a (ou pas) des slash dans l'URL de base et/ou le chemin fourni, du genre : 

```python
    if not path.startswith("/"):
        path = "/" + path
```

(et pareil pour l'URL de base)